### PR TITLE
Persian Translation

### PR DESCRIPTION
**Pull request by sallar**

### CODE REVIEW

File: locales/fa.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  "new-group": "گروه جدید",
+  "title-asc": "عنوان: صعودی",
+  "title-desc": "عنوان: نزولی",
+  "time-asc": "زمان: صعودی",
+  "time-desc": "زمان: نزولی",
+  "add-group": "افزودن گروه",
+  "rename": "تغییر نام",
+  "move-to-root": "انتقال به ریشه",
+  "move-to-group": "انتقال به گروه",
+  "move-to-group-custom": "انتقال به گروه {title}",
```
Comment (perry-mitchell) [inline locales/fa.json]: Hmm yes I see.. 🤔

File: locales/fa.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  "username": "نام کاربری",
+  "secure-password": "رمز عبور امن",
+  "untitled": "بدون عنوان",
+  "add-new-field": "افزودن فیلد جدید",
+  "copy": "کپی",
+  "hide": "مخفی",
+  "reveal": "نمایش دادن",
+  "reveal-password": "نمایش رمزعبور",
+  "installing": "در حال نصب",
+  "update-available-message": "باترکاپ {version} آماده است. برای نصب اینجا کلیک کنید.",
```
Comment (perry-mitchell) [inline locales/fa.json]: Uh huh.. Ok. That makes sense.

File: locales/fa.json [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  "date": "تاریخ",
+  "open-url-in-browser": "باز کردن لینک در مرورگر",
+  "new-field": "فیلد جدید",
+  "label": "عنوان",
+  "archive": "آرشیو",
+  "file": "پروند",
+  "archive-not-found-error": "آرشیو پیدا نشد.",
+  "authentication-failed-error": "در تایید اعتبار خطایی رخ داد.",
+  "unknown-error": "یک خطای ناشناخته رخ داد.",
+  "passwords-font-match-error": "رمز عبور شما صحیح نیست.",
```
Comment (perry-mitchell) [inline locales/fa.json]: Yes yes.. 👌

File: locales/fa.json [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  "entry-inputs-empty-info": "لطفا حداقل یک عنوان وارد کنید.",
+  "entry-title-empty-info": "لطفا یک عنوان وارد کنید.",
+  "entry-not-found-error": "سند پیدا نشد.",
+  "buttercup-archives": "آرشیوهای باترکاپ",
+  "load-a-buttercup-archive": "خواندن از یک آرشیو",
+  "create-a-new-buttercup-archive": "ساخت یک آرشیو جدید",
+  "invalid-import-type-requested-error": "درخواست درون‌ریزی پرونده نامرتبط",
+  "open-file-for-importing-error": "برای عملیات درون‌ریزی نیاز است که پنجره اصلی باترکاپ باز باشد.",
+  "yes": "بله",
+  "no": "خیر",
```
Comment (perry-mitchell) [inline locales/fa.json]: Are you sure on this one @sallar? 😬

**Comment by sallar** [comment]
@amin3mej Thanks for the great work. This is now going into master.

**Pull request merged**